packages/sanity/src/tasks/src/tasks/components/activity/TaskActivityEditedAt.tsx

```diff
@@ -0,0 +1,64 @@
+import {DotIcon} from '@sanity/icons'
```

Comment:
This file will be updated by https://github.com/sanity-io/sanity/tree/feat/edx-1137 as a fast follow after this is merged to next